cli/src/pcluster/validators/fsx_validators.py

```diff
@@ -222,3 +222,16 @@ def _validate(self, auto_import_policy, import_path):
             bucket = get_bucket_name_from_s3_url(import_path)
             if AWSApi.instance().s3.get_bucket_region(bucket) != get_region():
                 self._add_failure("FSx auto import is not supported for cross-region buckets.", FailureLevel.ERROR)
+
+
+class FsxDescribeVolumesValidator(Validator):
```

Comment:
The goal of the PR is to implement a unit test to prove that when the user does not have the permissions to describe the volumes, then the generation of the cluster template would fail.

Why are we adding a new validator?

Reply:
This approach is much more straightforward than attempting to mock the failure of `describe_volumes` from within `test_shared_storage.py`

Reply:
this was removed